src/utils/global/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+
+import styles from './tooltip.module.scss'
+
+interface TooltipProps {
+  text: string;
+  children: React.ReactNode;
+  title: string;
+  textContainerWidth?: string;
+  leftOffset?: string;
+}
+
+const Tooltip: React.FC<TooltipProps> = (props) => {
+  const [show, setShow] = useState(false);
+
+  const tooltipStyle: React.CSSProperties = {
+    opacity: show ? 1 : 0,
+    width: props.textContainerWidth ? props.textContainerWidth : "max-content",
+    left: props.leftOffset,
+  };
+
+  return (
+    <>
+      <div
+        className={styles.wrapper}
```

Comment:
Per my other comments/suggestions, I think this should also accept an optional className prop that is passed in here -- we have a util that'll do that cleanly, `classes`

Reply:
> something with the passing the className as a prop, would the tooltip component have access to those styles through its module.scss? I'm not sure if it's setup using the .module.scss file extension.

Yeah it should work just fine